packages/micropip/test_micropip.py

```diff
@@ -810,3 +814,65 @@ async def run_test(selenium, url, wheel_name):
             ]
 
         run_test(selenium_standalone_micropip, url, SNOWBALL_WHEEL)
+
+
+@contextmanager
+def does_not_raise():
+    yield
+
+
+def raiseValueError(msg):
+    return pytest.raises(ValueError, match=msg)
+
+
+@pytest.mark.parametrize(
+    "interp, abi, arch,ctx",
+    [
+        (
+            "cp35",
+            "cp35m",
+            "macosx_10_9_intel",
+            raiseValueError(
+                "Wheel platform 'macosx_10_9_intel' .* Pyodide's platform 'emscripten_3_1_14_wasm32'"
+            ),
+        ),
+        (
+            "cp35",
+            "cp35m",
+            "emscripten_2_0_27_wasm32",
+            raiseValueError(
+                r"Emscripten v2.0.27 but Pyodide was built with Emscripten v3.1.14"
+            ),
+        ),
+        (
+            "cp35",
+            "cp35m",
+            "emscripten_3_1_14_wasm32",
+            raiseValueError(
+                "Wheel abi 'cp35m' .* Supported abis are 'abi3' and 'cp310'."
```

Comment:
Maybe remove Emscripten and Python version values from the regex check so that we don't have to update these value when we update Emscripten or Python version?